docs/reference/mapping.asciidoc

```diff
@@ -85,10 +85,12 @@ causing a mapping explosion:
 ====
 The limit is in place to prevent mappings and searches from becoming too
 large. Higher values can lead to performance degradations and memory issues,
-especially in clusters with a high load or few resources.
+especially in clusters with a high load or few resources. 
 
-If you increase this setting, we recommend you also increase the
-<<search-settings,`indices.query.bool.max_clause_count`>> setting, which
+You can have mapping with even more amount of fields, but searching by significant
```

Comment:
I think that an excess of fields is problematic for more than just searches. As mentioned elsewhere in this doc, it can lead to significant memory pressure, and indexing and merging also work more smoothly with smaller mappings. It is not the case that "each document field is the independent Lucene index" as you claim. I think this change will be misleading and would prefer that we keep the more general wording that exists today.

Reply:
"memory pressure" - for what operations?
"indexing and merging also work more smoothly with smaller mappings" - of course, the more indexable fields you have, the longest index time per document you would have, but it's not critical if you need this data in elasticsearch anyway, in one document or in a few separate documents.
And what operation do you name "merging"? you talking about one of search process steps or this is about segment merging?